keadm/cmd/keadm/app/cmd/edge/joinbeta.go

```diff
@@ -181,7 +181,11 @@ func createEdgeConfigFiles(opt *common.JoinOptions) error {
 	}
 	if edgeCoreConfig == nil {
 		klog.Infoln("The configuration does not exist or the parsing fails, and the default configuration is generated")
-		edgeCoreConfig = v1alpha1.NewDefaultEdgeCoreConfig()
+		cfgOpt := &v1alpha1.ConfigOptions{
+			ImageRepository: opt.ImageRepository,
+			KubeEdgeVersion: opt.KubeEdgeVersion,
+		}
+		edgeCoreConfig = v1alpha1.NewDefaultEdgeCoreConfig(cfgOpt)
```

Comment:
Why not modify the `edgeCoreConfig` at here with `opt.ImageRepository` and `opt.KubeEdgeVersion` directly?

Reply:
I think if we need to add other parameters to modify the configuration file in the future, it is more convenient to use a structure directly

Reply:
In futrue, if we need to add config field for edgecore, add it to edgecore's config directly. And if add the options for keadm, the config file for keadm is a better way, then converting the keadm config to edgecore. How do you think?

Reply:
> In futrue, if we need to add config field for edgecore, add it to edgecore's config directly. And if add the options for keadm, the config file for keadm is a better way, then converting the keadm config to edgecore. How do you think?

And also supports converting the edgecore.yaml to keadm.yaml is cool.

Reply:
That is a good idea.
But I don't think it should affect this PR. 
For example, we might discuss the contents of the keadm configuration file, default parameters, etc. 
This should be a process.